x-pack/plugins/siem/server/lib/timeline/saved_object.ts

```diff
@@ -254,12 +260,14 @@ export class Timeline {
     return {
       totalCount: savedObjects.total,
       timeline: timelinesWithNotesAndPinnedEvents.map(([notes, pinnedEvents, timeline]) =>
-        timelineWithReduxProperties(notes, pinnedEvents, timeline)
+        timelineWithReduxProperties(notes, pinnedEvents, timeline, userName)
       ),
     };
   }
 }
 
+const convertStringToBase64 = (text: string): string => new Buffer(text).toString('base64');
```

Comment:
This is a deprecated and dangerous Buffer call on the server side:

<img width="848" alt="Screen Shot 2019-06-10 at 8 02 44 PM" src="https://user-images.githubusercontent.com/1151048/59238308-c06c1f00-8bba-11e9-8034-a39a19bd5eb5.png">


You should replace it with `Buffer.from` and make a new PR to fix this.